bin/pg_bouncer

```diff
@@ -4,8 +4,12 @@ cd "$(dirname "$0")/.."
 
 if [ ! -f "bin/start-pgbouncer-stunnel" ]; then
   echo "warning: pgbouncer buildpack not found, setting PGBOUNCER_ENABLED=false"
-  exit
+  export PGBOUNCER_ENABLED=false
 fi
 
-export PGBOUNCER_PREPARED_STATEMENTS=false
-bin/start-pgbouncer-stunnel "$@"
+if [ "$PGBOUNCER_ENABLED" = '1' ] || [ "$PGBOUNCER_ENABLED" = 'true' ]; then
```

Comment:
I kinda decided I didn't want this extra opt-in in this application. If a process is prefixed with `pg_bouncer` then it should be used. :)